conf/tai.routes

```diff
@@ -165,6 +165,7 @@ POST        /update-income/bonus-overtime-amount/:empId
 ## Regular Income URIs
 GET         /update-income/cancel/:id                                                  controllers.IncomeController.cancel(id: Int)
 GET         /update-income/select-taxable-pay                                          controllers.IncomeController.viewIncomeForEdit
+GET         /test-only/update-income/delete/:id                                        controllers.IncomeController.delete(id: Int)
```

Comment:
why not just using the DELETE verb?

Reply:
Also this should be part of conf/testOnlyDoNotUseInAppConf.routes

Reply:
Get is to facilitate testers. The selenium web driver can easily call Get as its doing at other places. With Delete the testers will have to write custom code for this.